scvi/core/models/__init__.py

```diff
@@ -143,17 +230,16 @@ class RNASeqMixin:
     @torch.no_grad()
```

Comment:
The class must have a description as well. Contains all the primitives for xxxx

Reply:
can add these in the docstrngs for the backend PR